docs/ref/models/fields.txt

```diff
@@ -426,6 +426,27 @@ precedence when creating instances in Python code. ``db_default`` will still be
 set at the database level and will be used when inserting rows outside of the
 ORM or when adding a new field in a migration.
 
+If a field is configured with db_default but lacks a default, accessing the field
+on an unsaved model instance, where no value has been explicitly assigned,
+will return a DatabaseDefault object::
+
+    from django.db import models
+    from django.db.models.functions import Now
+
+
+    class MyModel(models.Model):
+        # No default provided at the Django (Python) level, only db_default
+        created = models.DateTimeField(db_default=Now())
+
+.. code-block:: shell
+
+    >>> instance = MyModel()
+    >>> print(instance.created)
+    <django.db.models.expressions.DatabaseDefault object at 0x743257147ca0>
+
+The actual value for the field is determined by the database when the model
+instance is saved.
```

Comment:
Been playing around with the sentence structure and I think the following is clear
```suggestion
If a field has a ``db_default`` without a :attr:`~Field.default` set and no
value is assigned, a ``DatabaseDefault`` object is returned as the field value
on unsaved model instances. The actual value for the field is determined by the
database when the model instance is saved.
```
If this is clear enough, then we won't need the example

Reply:
I mean it is clear enough, So should we get rid of the  examples above? As i feel it solves the mentioned issue in the ticket and can also be understood without using a very detailed example...

Reply:
I think so, yes